cli/options/options.go

```diff
@@ -239,7 +241,11 @@ func HandleLoggingParams(debug bool, cfg config.ApplicationConfiguration) (*zap.
 	cc.DisableStacktrace = true
 	cc.EncoderConfig.EncodeDuration = zapcore.StringDurationEncoder
 	cc.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
-	cc.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	if term.IsTerminal(int(os.Stdout.Fd())) {
+		cc.EncoderConfig.EncodeTime = zapcore.TimeEncoderOfLayout("")
```

Comment:
Zap doesn't support an empty time encoder, so it's a little hack and this code will produce indented logs. @roman-khimov, @532910, are we OK with the following output?
```
	INFO	MaxBlockSize is not set or wrong, setting default value	{"MaxBlockSize": 262144}
	INFO	MaxBlockSystemFee is not set or wrong, setting default value	{"MaxBlockSystemFee": 900000000000}
	INFO	MaxTransactionsPerBlock is not set or wrong, using default value	{"MaxTransactionsPerBlock": 512}
	INFO	MaxValidUntilBlockIncrement is not set or wrong, using default value	{"MaxValidUntilBlockIncrement": 5760}
	INFO	restoring blockchain	{"version": "0.2.12"}
```

Reply:
no

Reply:
@AliceInHunterland, can we remove the starting indent? From what I searched, we can't.

Reply:
We can do this:

```
INFO    initial gas supply is not set or wrong, setting default value   {"InitialGASSupply": "52000000"}
INFO    MaxBlockSize is not set or wrong, setting default value {"MaxBlockSize": 262144}
INFO    MaxBlockSystemFee is not set or wrong, setting default value    {"MaxBlockSystemFee": 900000000000}
INFO    MaxTransactionsPerBlock is not set or wrong, using default value        {"MaxTransactionsPerBlock": 512}

```